src/index.html

```diff
@@ -665,34 +665,102 @@ <h3 class="padding-bottom-large">FAQs</h3>
+                                                </p>
                                             </div>
                                         </li>
                                         <li class="accordion-item" data-accordion-item>
-                                            <a href="#collapse19" class="accordion-title"><h5>What new variant accessions will be administered by the EVA?</h5></a>
+                                            <a href="#collapse19" class="accordion-title"><h5>What variant accessions will be administered by the EVA?</h5></a>
                                             <div class="accordion-content" data-tab-content>
                                                 <p>
-                                                    EVA shall follow the same accessioning format and rules that are employed by dbSNP: non-human variants that are submitted to the EVA shall be administered ‘ss’ accessions and these shall periodically be clustered to form ‘rs’ accessions.
+                                                    The EVA shall follow the same accessioning principles employed by dbSNP: non-human variants that are submitted to the EVA are issued 'Submitted SNP' (ss) accessions, and these are periodically clustered to form 'Reference SNP' (rs) accessions.
                                                 </p>
                                             </div>
                                         </li>
                                         <li class="accordion-item" data-accordion-item>
                                             <a href="#collapse20" class="accordion-title"><h5>Will it be possible to retrieve existing dbSNP accessions from the EVA?</h5></a>
                                             <div class="accordion-content" data-tab-content>
                                                <p>
-                                                Yes. The EVA is committed to the continuation of existing dbSNP ‘submitted SNP’ (ss) and ‘reference SNP’ (rs) accessions. These accessions shall be retrievable via the
-                                                <a href="?Variant%20Browser&species=btaurus_umd31&selectFilter=region&studies=PRJEB6119%2CPRJEB7061%2CPRJEB14878&region=1%3A3000000-3100000">EVA variant browser</a> and
-                                                <a href="?API">web services API</a>
+                                                   Yes. The EVA is committed to the continuation of existing dbSNP ‘submitted SNP’ (ss) and ‘reference SNP’ (rs) accessions.
+                                               </p>
+                                               <p> The accessions of those dbSNP variants that satisfy the EVA submission requirements will be retrievable via the
+                                                    <a href="?Variant%20Browser&species=hsapiens_grch37&selectFilter=snp&snp=rs55880202">EVA variant browser</a> and

```

Comment:
This example was probably not very good (my fault), as we should prefer non-human species here. Please replace it with

```
?Variant%20Browser&species=mmusculus_grcm38&selectFilter=snp&snp=rs585444580
```